src/components/Modal/README.md

```diff
@@ -26,7 +26,9 @@ React.createElement(() => {
   const modalPrimaryButtonRef = React.useRef();
   const modalCloseButtonRef = React.useRef();
   return (
-    <>
+    <RUIProvider globalProps={{
+      Modal: { preventScrollUnderneath: window.document.documentElement }
```

Comment:
I might miss something, but why is everywhere `window.document.documentElement`? Why not to use default?

Reply:
Because default only works in cases where the `<body>` element is scrollable. Our documentation platform (MkDocs material) however scrolls the `<html>` element.

From what @adamkudrna was saying I understand that scrolling `<body>` is more common, but it is not the only valid way. The API tweaks in this PR reflect that I think.